SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialWorkerFlags.h

```diff
@@ -31,8 +32,9 @@ class SPATIALGDK_API USpatialWorkerFlags : public UBlueprintFunctionLibrary
 	static void UnbindFromOnWorkerFlagsUpdated(const FOnWorkerFlagsUpdatedBP& InDelegate);
 
 	static FOnWorkerFlagsUpdated OnWorkerFlagsUpdated;
+
+	static void ApplyWorkerFlagUpdate(const Worker_FlagUpdateOp& Op);
```

Comment:
We should remove the friend if changing the access modifier.

Reply:
I decided to move it back, didn't notice the friend below at first.

Reply:
However is this meant to be part of the public interface. Generally this seems like the kind of thing that can be reasonably written without using friends but do we want this to be public?